### PR TITLE
magicrescue 1.1.9 (new formula)

### PR DESCRIPTION
**Pull request by apfohl**
This PR add's a new formular to Homebrew for Magic Rescue.

Magic Rescue scans a block device for file types it knows how to recover and calls an external program to extract them. It looks at "magic bytes" in file contents, so it can be used both as an undelete utility and for recovering a corrupted drive or partition. As long as the file data is there, it will find it.

### CODE REVIEW

File: Library/Formula/magicrescue.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Magicrescue < Formula
+  desc "Magic Rescue extracts files of known types from a block device."
+  homepage "http://www.itu.dk/people/jobr/magicrescue/"
```
Comment (DomT4) [inline Library/Formula/magicrescue.rb]: You can use `https://` for both these links.

File: Library/Formula/magicrescue.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  url "http://www.itu.dk/people/jobr/magicrescue/release/magicrescue-1.1.9.tar.gz"
+  sha256 "a920b174efd664afe9760a43700588c9c5e6182cb13d7421e07ab613bceeb3c7"
+
+  def install
+    system "./configure", "PREFIX=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
```
Comment (DomT4) [inline Library/Formula/magicrescue.rb]: Please add a test :). See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by bfontaine** [comment]
The build fails because it tries to installs stuff directly under `/usr/local/share` and `/usr/local/man`. You can use the special variables `share` and `man` to tell it where these directories are located. Everything is installed under a common prefix; things later get symlinked under `/usr/local/share` and `/usr/local/man`.

File: Library/Formula/magicrescue.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  end
```
Comment (bfontaine) [inline Library/Formula/magicrescue.rb]: Please add a test here.

**Comment by dunn** [comment]
```
==> audit problems
magicrescue:
 * A top-level "man" directory was found
Homebrew requires that man pages live under share.
This can often be fixed by passing "--mandir=#{man}" to configure.
```

**Comment by apfohl** [comment]
I'm working on it.

**Comment by bfontaine** [comment]
Anything new here?

**Comment by apfohl** [comment]
Sorry, still not sure how to solve this. After christmas I can look at this more closely.

File: Library/Formula/magicrescue.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  url "http://www.itu.dk/people/jobr/magicrescue/release/magicrescue-1.1.9.tar.gz"
+  sha256 "a920b174efd664afe9760a43700588c9c5e6182cb13d7421e07ab613bceeb3c7"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+  end
```
Comment (mikemcquaid) [inline Library/Formula/magicrescue.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/magicrescue.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class Magicrescue < Formula
+  desc "Magic Rescue extracts files of known types from a block device."
```
Comment (bfontaine) [inline Library/Formula/magicrescue.rb]: You might want to write a `desc` that doesn’t start with “Magic Rescue”. `brew desc magicrescue` will show the following:
```
magicrescue: Magic Rescue extracts files of known types from a block device.
```

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit with comments addressed if desired. Thank for your submission to Homebrew!

**Pull request closed without merging**